projects/maps-ng/src/components/si-map/shared/themes/style.ts

```diff
@@ -0,0 +1,2905 @@
+/**
+ * Copyright (c) Siemens 2016 - 2025
+ * SPDX-License-Identifier: MIT
+ */
+
+// Fonts
+// TODO: This is a workaround to disable loading of google fonts.
+const italicFonts = ['Siemens Sans'];
+const regularFonts = ['Siemens Sans'];
+const boldFonts = ['Siemens Sans'];
+
+// Colors
+const siemensDeepBlue900 = 'rgba(32, 33, 50, 1)';
+const siemensGreen700 = (opacity: number): string => `rgba(28, 112, 63, ${opacity})`;
+const siemensGreen500 = (opacity: number): string => `rgba(40, 191, 102, ${opacity})`;
+const siemensGreen300 = (opacity: number): string => `rgba(114, 230, 163, ${opacity})`;
+const siemensBlue700 = 'rgb(30, 82, 153)';
+const siemensBlue100 = 'rgb(210, 226, 247)';
+
```

Comment:
I think we need to consumer all of those from the css vars.
Same for every location in this file

Reply:
I am not sure how to handle this as some of the rgb are not part of our tokens. e.g `siemensDeepBlue900` here is #202132 which is no where in our tokens. 🤷